tests/puppeteer.test.js

```diff
@@ -0,0 +1,15 @@
+// The initial page loads takes ~7 seconds on my machine.
```

Comment:
Is that just the first page load, or are we looking at an execution time of ~7 seconds for every test that's being run?

Reply:
It's the time taken for the server to start up - the tests themselves are a few milliseconds.